src/main/java/pl/simpleascoding/tutoringplatform/exception/UserIsNotATeacherException.java

```diff
@@ -0,0 +1,13 @@
+package pl.simpleascoding.tutoringplatform.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.UNAUTHORIZED)
+public class UserIsNotATeacherException extends IllegalArgumentException {
+
+    public UserIsNotATeacherException() {
+        super("User is not a teacher");
```

Comment:
Please move this string to a static, final field, like this: `private static final String MESSAGE_USER_IS_NOT_A_TEACHER = "User is not a teacher"`